src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() { return false; }
```

Comment:
Having a default implementation of `isMutating()` in the parent class is good, as this reduces code. However, giving an arbitrary return value (E.g. true or false) reduces code understandability. Think about how refactoring to an `abstract` class with an  `abstract` method might work in this scenario.

Reply:
I understand. Thank you.